ob1k-consul/src/main/java/com/outbrain/ob1k/consul/ConsulHealth.java

```diff
@@ -15,6 +15,8 @@
 
   ComposableFuture<List<HealthInfoInstance>> filterDcLocalHealthyInstances(final String service, final String filterTag);
 
+  ComposableFuture<List<HealthInfoInstance>> filterDcHealthyInstances(final String dc, final String service, final String filterTag);
```

Comment:
there's importance to fields order when using path params - service should be first, then dc, else not gonna work.